hamming.js

```diff
@@ -1,17 +1,11 @@
-//
-// This is only a SKELETON file for the 'Hamming' exercise. It's been provided as a
-// convenience to get you started writing code faster.
-// Make sure to look at test.script.js--that should give you some hints about what is
-// expected here.
-
-'use strict';
-
 function Hamming() {};
-
 Hamming.prototype.compute = function (strand1, strand2) {
-//
-// YOUR CODE GOES HERE
-//
-};
+var diff = 0;
+for (i = 0; i <strand1.length; i++) {
+  if(strand1[i] != strand2[i]) {
+diff = diff + 7;
+  }
+ }
+return diff;
 
-module.exports = Hamming;
```

Comment:
sorry

On Mon, Apr 25, 2016 at 4:48 PM, Yvan Sraka <notifications@github.com>
wrote:

> In hamming.js
> <https://github.com/simplonco/hamming/pull/2#discussion_r60925456>:
>
> >
> > -module.exports = Hamming;
>
> *No No No No [image: :rage:] !*
>
> Do *not* remove this line, if you want tests work:
>
> module.exports = Hamming;
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/simplonco/hamming/pull/2/files/73ab794280dc2149e2b39e2aa8f94ebecb5c3a5c#r60925456>
>